R/pkg/R/mllib_classification.R

```diff
@@ -338,7 +338,7 @@ setMethod("spark.logit", signature(data = "SparkDataFrame", formula = "formula")
               if (!is.null(lowerBoundsOnCoefficients) && (row != nrow(upperBoundsOnCoefficients)
                 || col != ncol(upperBoundsOnCoefficients))) {
                 stop(paste0("dimension of upperBoundsOnCoefficients ",
-                           "is not the same as lowerBoundsOnCoefficients", sep = ""))
+                            "is not the same as lowerBoundsOnCoefficients"))
```

Comment:
This space seems a mistake.

Reply:
it's aligning to the `"` on the previous line. I can revert if you want to minimize the diff

Reply:
oh, that's true. I misread.